src/plugins/intel_gpu/src/kernel_selector/kernels/rms/rms_kernel_bfyx_opt.cpp

```diff
@@ -8,6 +8,24 @@
 
 namespace kernel_selector {
 static constexpr size_t subgroup_size = 16;
+static constexpr size_t max_items_num = 8;
+
+// Compute maximum possible LWS that does not exceed device capabilities and optimizes number of global memory reads
+std::pair<int, int> get_item_num_and_lws(const rms_params params, size_t data_size);
+std::pair<int, int> get_item_num_and_lws(const rms_params params, size_t data_size) {
+    size_t lws = 1;
+    size_t itemsNum = data_size;
+    const auto& input = params.inputs[0];
+    auto local_mem_per_wi = 2 * BytesPerElement(input.GetDType());
+    auto max_lws = std::min(params.engineInfo.maxWorkGroupSize, params.engineInfo.maxLocalMemSize / local_mem_per_wi);
+
+    while ((itemsNum > 8 || lws < itemsNum) && (2 * lws <= max_lws)) {
```

Comment:
use max_items_num here instead of 8

Reply:
Since 'max_items_num' is only used one time in current version, I cleaned it now.